### PR TITLE
Add new RIG_MODE_STEREOSCOPIC_INTERLACED mode for 3d monitors

### PR DESCRIPTION
**Pull request by bilkusg**
As per discussion in the forum, I have created a pull request with the changes I made to allow support of 3d interlaced monitors in stereo mode

### CODE REVIEW

File: src/Shaders/stereoscopicInterlace.fragment.fx [hunk 1]
```diff
@@ -4,9 +4,11 @@ varying vec2 vUV;
 uniform sampler2D camASampler;
 uniform sampler2D textureSampler;
 uniform vec2 stepSize;
+// Updated for STEREOSCOPIC_INTERLACED
```
Comment (deltakosh) [inline src/Shaders/stereoscopicInterlace.fragment.fx]: not required

File: src/Shaders/stereoscopicInterlace.fragment.fx [hunk 2]
```diff
@@ -16,22 +18,34 @@ void main(void)
     
 #ifdef IS_STEREOSCOPIC_HORIZ
 	    useCamB = vUV.x > 0.5;
+        useCamA = !useCamB;
 	    texCoord1 = vec2(useCamB ? (vUV.x - 0.5) * 2.0 : vUV.x * 2.0, vUV.y);
 	    texCoord2 = vec2(texCoord1.x + stepSize.x, vUV.y);
+#else
+#ifdef IS_STEREOSCOPIC_INTERLACED
+	    float rowNum = floor(vUV.y / stepSize.y);
+        useCamA = mod(rowNum,2.0) == 1.0;
```
Comment (deltakosh) [inline src/Shaders/stereoscopicInterlace.fragment.fx]: indentation seems off

Comment (bilkusg) [reply src/Shaders/stereoscopicInterlace.fragment.fx]: Have updated file with correct indentation. I seem to have had a problem with tab settings in vscode.....

**Comment by sebavan** [comment]
@deltakosh will be back tomorrow to approve this one, I did not follow it from the beginning so prefere to wait until then.

**Pull request merged**